src/geo/lng_lat.js

```diff
@@ -10,6 +14,25 @@ import LngLatBounds from './lng_lat_bounds.js';
 */
 export const earthRadius = 6371008.8;
 
+/*
+*/
+export const globeRadius = EXTENT / Math.PI / 2.0;
+
+export function latLngToEcef(lat: number, lng: number, radius: number = globeRadius): [number, number, number] {
```

Comment:
Could we consolidate with the same function in https://github.com/mapbox/mapbox-gl-js/blob/ad9c713939882f23db7902361d1d34a96280def6/src/geo/projection/globe_util.js#L439